.github/workflows/isolated.yml

```diff
@@ -0,0 +1,133 @@
+name: The Droids You're Looking For
```

Comment:
We should probably remove this for the merge since it's for testing the CI.

Reply:
done